pastas/model.py

```diff
@@ -59,8 +59,9 @@ class Model:
     constant: bool, optional
         Add a constant to the model (Default=True).
     noisemodel: bool, optional
-        Add the default noisemodel to the model. A custom noisemodel can be added
-        later in the modelling process as well.
+        When true, add the default noisemodel to the model. A custom noisemodel can be
```

Comment:
Should we not set the default to None, and then if True or False is provided, raise an Error that this argument is deprecated and 1) no noisemodel is added by default anymore (no need for False), and 2) a noisemodel should be added by using Model.add_noisemodel() ? That way people can use 1.5 to clean up their code (like we did in the examples) to match the new pattern to adding a noise model.